packages/engine/src/worker/runner/javascript/mini_v8/function.rs

```diff
@@ -104,7 +104,7 @@ fn manually_drop_arg_descs(arg_descs: Vec<ValueDesc>) {
     // Ownership of the arguments' value pointers was taken by C++. They've already been properly
     // dropped:
     for desc in arg_descs.into_iter() {
-        ManuallyDrop::new(desc);
+        mem::forget(desc);
```

Comment:
Can we double check this is correct. **If** the "value pointer" is taken by C++ (I have not checked if this is true or what exactly is taken), do we need to worry about the rest of the `ValueDesc` struct? If it's still cleared up with this then no worries

Reply:
In any case, `mem::forget` is defined like this:
```rust
pub const fn forget<T>(t: T) {
    let _ = ManuallyDrop::new(t);
}
```

Reply:
Yeah well at least this isn't a regression, just making sure we aren't covering a potential bug